### DISCUSSION TITLE
Devices have 'failed to set up' status after power outage

### DISCUSSION THREAD

**Discussion by roja45** [make-all/tuya-local#1118] [Q&A]
Hello,

After a power outage, a good selection of the tuya devices have a 'failed to set up' state in HA, the entity is unavailable in HA, but they are online in IOT and clicking the reload option on each one brings them straight back.

Do you have any idea why this is happening?

![image](https://github.com/make-all/tuya-local/assets/19314911/5baf4d4d-f42c-45bc-a4e9-ffb06ac6c486)

**Comment by make-all** [comment]
If you set the protocol and IP address in the config, they should recover quicker.
